src/main/java/org/fundacion/jala/converter/controller/AudioConverterController.java

```diff
@@ -51,27 +52,30 @@
      * @param checksum is checksum of audio.
      * @param metadata if metadata is extracted from the audio.
      * @return path to download files.
-     * @throws IOException  is exception when invalid path.
-     * @throws InterruptedException  is exception if process is interrupted.
+     * @throws IOException is a exception when invalid input is provided.
+     * @throws InterruptedException is exception if process is interrupted.
      */
     @PostMapping("/convertAudio")
-    public String uploadFile(final @RequestParam("file") MultipartFile file, final @RequestParam("format") String format,
-                             final @RequestParam("bitrate") String bitrate, final @RequestParam("volume") String volume,
+    public String uploadFile(final @RequestParam("file") MultipartFile file,
+                             final @RequestParam("format") String format,
+                             final @RequestParam("bitrate") String bitrate,
+                             final @RequestParam("volume") String volume,
                              final @RequestParam("hz") String hz,
                              final @RequestParam("audiochannel") String audioChannel,
                              final @RequestParam("checksum") String checksum,
-                             final @RequestParam("metadata") String metadata)
-            throws IOException, InterruptedException {
+                             final @RequestParam("metadata") String metadata) throws IOException, InterruptedException {
+        final String baseUrl;

```

Comment:
I don't understand why this parameter is final?

Reply:
because it is a suggestion of the checkstyle tool.